src/libraries/System.Speech/src/SRID.cs

```diff
@@ -0,0 +1,406 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Speech
+{
+    internal enum SRID
```

Comment:
We had a way to generate these in buildtools, but I believe it was dropped when unifying in arcade: https://github.com/dotnet/buildtools/pull/2086

Not something you need to change now, but perhaps a future cleanup.

Reply:
Yup although this one's particularly fiddly as the order has to match the native header.

Reply:
IIRC this was specifically the pattern I added that enum for and preserved the ordering from Resx in the enum.  From the PR:
```
resourceId = (ResourceEnumType) (ResourceEnumType.FirstErrrorId + (errorCode - firstErrorCode))
```

Reply:
That said, if we aren't likely to change this code, it doesn't need to be run through a generator.  Perhaps a mitigation would be a test case that ensures the enum matches the resource IDs?